pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -0,0 +1,202 @@
+        // },
+        {
+          name:  'containerName',
+          label: this.t('containers.manage.table.header.containerName'),
+          sort:  ['containerName', 'image', 'imageName'],
+        },
+        {
+          name:  'imageName',
+          label: this.t('containers.manage.table.header.image'),
+          sort:  ['imageName', 'containerName', 'imageName'],
+        },
+        {
+          name:  'ports',
+          label: this.t('containers.manage.table.header.ports'),
+          sort:  ['ports', 'containerName', 'imageName'],
+        },
+        {
+          name:  'started',
+          label: this.t('containers.manage.table.header.started'),
+          sort:  ['si', 'containerName', 'imageName'],
+          width: 120,
+        },
+      ],
+    };
+  },
+  computed: {
+    rows() {
+      if (!this.containersList) {
+        return [];
+      }
+
+      const containers = cloneDeep(this.containersList);
+
+      return containers
+        .map((container) => {
+          container.state = container.State;
+          container.containerName = container.Names[0].replace(/_[a-z0-9-]{36}_[0-9]+/, '');
+          container.started = container.Status;
+          container.imageName = container.Image;
+
+          container.availableActions = [
+            {
+              label:   'Stop',
+              action:  'stopContainer',
+              enabled: true,
+              icon:    'icon icon-pause',
+            }, {
+              label:      this.t('images.manager.table.action.delete'),
+              action:     'deleteContainers',
+              enabled:    true,
+              icon:       'icon icon-delete',
+              bulkable:   true,
+              bulkAction: 'deleteContainers',
+            },
+          ];
+
+          if (!container.stopContainer) {
+            container.stopContainer = this.stopContainer.bind(this, container);
+          }
+
+          if (!container.deleteContainers) {
+            container.deleteContainers = this.deleteContainers.bind(this, container);
+          }
+
+          return container;
+        });
+    },
+  },
+
+  mounted() {
+    this.$store.dispatch('page/setHeader', {
+      title:       this.t('containers.title'),
+      description: '',
+    });
+
+    // INFO: We need to set hasContainers outside of the component in the global scope so it won't re-render when we get the list.
+    const intervalId = setInterval(() => {
+      if (hasContainers) {
+        return;
+      }
+
+      if (window.ddClient) {
+        window.ddClient?.docker.listContainers().then((containers) => {
+          console.log(' containers;', containers);
+          // INFO: This will only get the currently running containers.
+          this.containersList = containers;
+          hasContainers = true;
+          this.mountTable = true;
+        });
+
+        clearInterval(intervalId);
+      }
+    }, 1000);
+  },
+  beforeDestroy() {
+    clearInterval(this.intervalId);
```

Comment:
I don't think this will clear any interval because there is no `this.intervalId` in this component.